packages/redux/src/returns/selectors.ts

```diff
@@ -165,3 +166,27 @@ export const areReturnPickupCapabilitiesLoading = (state: StoreState) =>
  */
 export const getReturnPickupCapabilitiesError = (state: StoreState) =>
   getPickupCapabilities(state.returns as ReturnsState).error;
+
+/**
+ * Returns the loading status for the get reschedule request.
+ *
+ * @function
+ *
+ * @param {object} state - Application state.
+ *
+ * @returns {boolean} Loading status.
+ */
+export const isPickupRescheduleRequestsLoading = (state: StoreState) =>
```

Comment:
Can you change it to `arePickupRescheduleRequestsLoading`?

Reply:
done!